nixos/maintainers/scripts/azure/README.md

```diff
@@ -0,0 +1,79 @@
+# nixos/azure
+
+#### Create & Upload "Official" Images
+
+1. Update `<nixpkgs>/nixos/modules/virtualisation/azure-images.nix`, as appropriate.
+2. From `<nixpkgs>/nixos/maintainers/scripts/azure`, run `./images-ensure.vhd`. This will ensure all images are built and re-upload them all.
+3. You should manually update `azure-images.nix` with the uploaded image URLs.
+
+```bash
+nvim ./../../../modules/virtualisation/azure-images.nix
+export AZURE_SUBSCRIPTION_ID="<production NixOS subscription ID>"
+export AZURE_RESOURCE_GROUP="NIXOS_PRODUCTION"
+export AZURE_STORAGE_ACCOUNT="nixosofficial"
```

Comment:
Will need to update these when we get the subscription info.
The official account might be a legacy SA in a non-RG, in which case we can try to move it or maybe just make a new "official" storage account, especially since I didn't pull over the very old images anyway.